test/lib/common_test.js

```diff
@@ -837,6 +838,33 @@ exports.postgresql = {
   }
 }
 
+// ElephantSQL
+exports.elephantsql = {
+  itCreatedServiceFilesWithExpectedContent: function (serviceName, serviceCredentials, servicePlan) {
+    var description = 'elephantsql'
+    var mapping = 'elephant_sql'
+    var label = 'elephantsql'
+    var plan = servicePlan || helpers.getBluemixDefaultPlan('elephantsql')
+    var sourceFile = 'ServiceElephantsql.swift'
```

Comment:
'ServiceElephantSql.swift' - service-enablement camelcases the laters after the `-`.